app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, counter_cache: true
+
+  # def self.update_post_comments_counter(comment)
+  #   post = comment.post
+  #   post.update(comments_counter: post.comments.counter)
+  # end
```

Comment:
- [ ] As already pointed out above, let's kindly get rid of commented codes. If you needs the previous version of your code, you can use git history for that specific